app/models/open_conference_ware/user.rb

```diff
@@ -256,6 +256,14 @@ def sessions
       return self.proposals.confirmed
     end
 
+    # For use in merging duplicate user records
+    def take_associations_from(dup)
+      dup.authentications.each { |a| self.authentications << a }
```

Comment:
Cool, thanks. I've updated those lines. Not sure if I can change the `dup.proposals.each` line though, is there a way to remove the speaker from the proposals without looping?